packages/generator/ni_measurement_plugin_sdk_generator/client/_support.py

```diff
@@ -148,6 +153,11 @@ def get_configuration_and_output_metadata_by_index(
 
     output_parameter_list = []
     for output in metadata.measurement_signature.outputs:
+        if output.message_type and output.message_type != "ni.protobuf.types.DoubleXYData":
+            raise click.ClickException(
+                f"Measurement outputs do not support {output.message_type}. Only DoubleXYData is supported."
+            )
```

Comment:
Ok, I think this should prevent `TypeError: Couldn't build proto file into descriptor pool` but I haven't tested it yet.

Reply:
Yes, both the above handling and https://github.com/ni/measurement-plugin-python/pull/984/files/78b383f49f860c8c2a4ba923c301790fe8301339#:~:text=.message_type%3A-,raise%20click.ClickException(,-f%22Measurement%20configurations handling will prevent the TypeError. I’ve also verified this with the waveform measurement in .NET.